lm_eval/tasks/exams/exams_openbook/_crosslingual_bg.yaml

```diff
@@ -0,0 +1,5 @@
+"dataset_name": "crosslingual_bg"
```

Comment:
Does having an open book version of the split WITHOUT the paragraphs make sense here?

Reply:
Oh that would just be the closed book version, see file lm_eval/tasks/exams/exams_closed_book/_crosslingual_bg.yaml.